.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Can you elaborate on this?

Reply:
seems to be something auto-generated during husky setup